### PR TITLE
Created hitl session module (controller, repository, service, model and routes) and all comments resolved

### PR DESCRIPTION
**Pull request by divah21**

### CODE REVIEW

**Review by DigiSenseiZim — CHANGES_REQUESTED** [review]
Apply all the changes in the comments, test then push. On cloning your repo and adding my own `.env` in a similar environment I was unable to fire the API up.

File: src/controllers/hitlMessagesController.js
[the remark this answers is not in the document]
Comment (divah21) [reply src/controllers/hitlMessagesController.js:1]: all comments resolved in HitlMessageController

File: src/repositories/hitlMessagesRepository.js
[the remark this answers is not in the document]
Comment (divah21) [reply src/repositories/hitlMessagesRepository.js:1]: comments addressed

File: src/controllers/hitlSessionController.js
[the remark this answers is not in the document]
Comment (divah21) [reply src/controllers/hitlSessionController.js:1]: comments resolved

**Review by DigiSenseiZim — APPROVED** [review]

**Review by DigiSenseiZim — APPROVED** [review]

**Pull request merged**